apiary.apib

```diff
@@ -2104,6 +2104,38 @@ Notifies API that a reconciliation file was uploaded, so that it can be processe
 
 Resources to fetch the links to download the reports created in our system.
 
+## Index [/reports]
+
+Returns the reports of the current client.
+
+### reports [GET]
+
++ Request (application/json)
+
+    + Header
+
+            Accept: application/json
+            Authorization: eyJ0eXAiOiJKV1QiLCJhbiAoOiJIUzI1NiJ9.eyJvX2F1dGhfYXBwbGljYXRpb25faWQiOjF9.h7ZNPpbd_Bu8yL27fmCAI_wgyZJI44b5eRRxwWUcwaQ
```

Comment:
only being authorized is needed to do a GET to this controller? doesn't it have a `:page` parameter? or it responds all reports available?